### PR TITLE
Neural net

### PR DESCRIPTION
**Pull request by mcbottcher**
should be able to run the neural network form the training window...
The python script also outputs a dictionary file mapping the files to the network output which can be used by the predictor

### CODE REVIEW

File: code/Qt/TrainWindow.cpp [hunk 1]
```diff
@@ -54,19 +54,26 @@ TrainWindow::TrainWindow(int16_t* dataPtr){
     trainGroup->setLayout(trainLayout);
 
 
+    netGroup = new QGroupBox(tr("Neural Network"));
 
-    homeButton = new QPushButton("Save");
-    connect(homeButton, &QPushButton::clicked, [this](){this->closeWindow();});
+    nettrainButton = new QPushButton("Run Network");
+    connect(nettrainButton, &QPushButton::clicked, [this](){this->startNeuralNet();});
```
Comment (jamielee183) [inline code/Qt/TrainWindow.cpp:60]: do we want to close the window after training? have a button so we can safely close the window

File: code/Qt/TrainWindow.cpp [hunk 2]
```diff
@@ -83,8 +90,18 @@ void TrainWindow::trainingDataLoop(){
...
+    QList<QListWidgetItem *> selected_movements = movementList->selectedItems();
+    
+    std::string command = "python3 neuralNet.py ";
+    
+    for(int i=0; i<selected_movements.size(); i++){
+        command += (selected_movements[i]->text()).toStdString();
+        command += " ";
+    }
+    
+    system(command.c_str());
```
Comment (jamielee183) [inline code/Qt/TrainWindow.cpp:109]: starting a python environment within c might be better?
[Embedding Python in C++](https://www.codeproject.com/Articles/820116/Embedding-Python-program-in-a-C-Cplusplus-code)
not sure if it makes a difference,

Comment (mcbottcher) [reply code/Qt/TrainWindow.cpp:109]: yeah I agree, just couldn't get to work and want to test it quickly, but I will change it.
I think the advantage is that QT will know when the script is finished

File: code/Qt/TrainWindow.cpp (hunk 1 above)
Comment (mcbottcher) [reply code/Qt/TrainWindow.cpp:60]: done

**Pull request merged**